src/executor/stack/executor.rs

```diff
@@ -1150,6 +1150,13 @@ impl<'config, 'precompiles, S: StackState<'config>, P: PrecompileSet> Handler
 		capture
 	}
 
+	fn get_create_address(
```

Comment:
Had to add this as there was no way to calculate the address that would be created at the point the event is emitted in runtime.